databridge.examples/databridge.examples.kafka-jsonatamultiple-aas/src/test/java/org/eclipse/digitaltwin/basyx/databridge/examples/kafkajsonatamultipleaas/test/TestAASUpdater.java

```diff
@@ -53,95 +61,176 @@
 import org.eclipse.paho.client.mqttv3.MqttException;
 import org.eclipse.paho.client.mqttv3.MqttPersistenceException;
 import org.eclipse.paho.client.mqttv3.MqttSecurityException;
+import org.junit.AfterClass;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import kafka.server.KafkaConfig;
+import kafka.server.KafkaServer;
+import scala.Option;
 
 public class TestAASUpdater {
+	private static Logger logger = LoggerFactory.getLogger(TestAASUpdater.class);
 	private static AASServerComponent aasServer;
 	private static DataBridgeComponent updater;
 	private static InMemoryRegistry registry;
+	
+	private static KafkaServer kafkaServer;
+	private static String kafkaTmpLogsDirPath;
 
+	private static TestingServer zookeeper;
+	
 	protected static IIdentifier deviceAAS = new CustomId("TestUpdatedDeviceAAS");
 	private static BaSyxContextConfiguration aasContextConfig;
 
 	@BeforeClass
-	public static void setUp() throws IOException {
-		registry = new InMemoryRegistry();
+	public static void setUp() throws Exception {
+		configureAndStartKafkaServer();
+		configureAndStartAASServer();
+		configureAndStartUpdaterComponent();
+		
+	}
 
+	private static void configureAndStartAASServer() {
 		aasContextConfig = new BaSyxContextConfiguration(4001, "");
 		BaSyxAASServerConfiguration aasConfig = new BaSyxAASServerConfiguration(AASServerBackend.INMEMORY, "aasx/updatertest.aasx");
 		aasServer = new AASServerComponent(aasContextConfig, aasConfig);
-		aasServer.setRegistry(registry);
+		aasServer.setRegistry(registry = new InMemoryRegistry());
+		aasServer.startComponent();
+	}
+	
+	@AfterClass
+	public static void tearDown() throws IOException {
+		updater.stopComponent();
+		aasServer.stopComponent();
+		zookeeper.close();
+		aasServer.stopComponent();
```

Comment:
This should stop the kafkaServer and not the aasServer again

Reply:
so glad that we have a review process :D